terraform-plans/configs/charmed-openstack-exporter-snap_main.tfvars

```diff
@@ -16,7 +16,9 @@ templates = {
     source      = "./templates/github/snap_check.yaml.tftpl"
     destination = ".github/workflows/check.yaml"
     vars = {
-      runs_on = "[[ubuntu-22.04], [self-hosted, jammy, ARM64]]",
+      runs_on         = "[[ubuntu-22.04], [self-hosted, jammy, ARM64]]",
+      python_versions = "['3.10']",
+      project         = "charmed-openstack-exporter-snap"
```

Comment:
Do you mind to change the variable name to 'tics_project' to make its purpose clearer? thanks!